lib/SILOptimizer/Mandatory/DefiniteInitialization.cpp

```diff
@@ -1786,8 +1941,7 @@ bool LifetimeChecker::diagnoseReturnWithoutInitializingStoredProperties(
   return true;
 }
 
-bool LifetimeChecker::isRestrictedActorInitSelf(const DIMemoryUse& Use,
-                                            ActorInitKind *Kind) const {
+bool LifetimeChecker::isRestrictedActorInitSelf(ActorInitKind *Kind) const {
```

Comment:
I think this snuck in, but can you please use camelCase. This is just an old file that needs to be updated a little. New code should use camelCase.